corehq/apps/export/models/new.py

```diff
@@ -832,9 +832,14 @@ class Meta(object):
     @classmethod
     def wrap(cls, data):
         from corehq.apps.export.views.utils import clean_odata_columns
+        # This is a temporary solution
```

Comment:
Here's I'm simply ensuring the `split_multiselects` is false for geojson exports. This is a temporary solution until the geosjon form export is compatible with the `split_multiselects` option (there's a ticket in the backlog for this).

Reply:
Thanks for the detail on the change @Charl1996 , can you please put that as a comment in the code itself?
Along with the link to the ticket.

Reply:
https://github.com/dimagi/commcare-hq/pull/34281/commits/866d298211309154ed1eb5ff2d25967be85ca180